tests/statements/vacuum.rs

```diff
@@ -89,9 +89,10 @@ async fn test_vacuum_command() {
     // twice, but we can have two different partition IDs with same object storage ID
     // See https://github.com/splitgraph/seafowl/issues/5
     let orphans = vec![
-        FILENAME_1,
-        "7b4ddceb7b8ac1869de495c355fd957ce94a891a1770c7e540fce4e47cd25a0e.parquet",
-        "b3e703ac60edd787afcdafc07b8cbb1fd6f5eb6c83e32f4a92ab7c77773bb151.parquet",
+        // TODO: Why is the first one not FILENAME_1?
+        "4d2e6f43a0dc42cedd89fe3510f477654c12a305d171c6213d2aa854682e4ee4.parquet",
+        "84ba4647437c6dcdcc565e655ea1f6ea50aa957e498731088856a60dd0696f51.parquet",
+        "e0e13b9f480fd3f0ccb5753235075cdccd766eaad2299a99173bc4f99f5adb07.parquet",
```

Comment:
It's because the orphan code now sorts the output (because it runs `DISTINCT`), so it's actually `84ba4647437c6dcdcc565e655ea1f6ea50aa957e498731088856a60dd0696f51.parquet` that is FILENAME_1 and I started using [`assert_eq_unordered_sort`](https://docs.rs/assert_unordered/latest/assert_unordered/macro.assert_eq_unordered_sort.html) to compare them

```suggestion
        FILENAME_1,
        "4d2e6f43a0dc42cedd89fe3510f477654c12a305d171c6213d2aa854682e4ee4.parquet",
        "e0e13b9f480fd3f0ccb5753235075cdccd766eaad2299a99173bc4f99f5adb07.parquet",
```

Reply:
Ah right; thanks!